subsys/bluetooth/services/fast_pair/include/fp_keys.h

```diff
@@ -85,6 +85,37 @@ int fp_keys_encrypt(const struct bt_conn *conn, uint8_t *out, const uint8_t *in)
  */
 int fp_keys_decrypt(const struct bt_conn *conn, uint8_t *out, const uint8_t *in);
 
+/** Encode data to Additional Data packet for the Fast Pair Seeker.
+ *
+ * Length of buffer used to store Additional Data packet must be at least equal to sum of
+ * @ref FP_CRYPTO_ADDITIONAL_DATA_HEADER_LEN and data_len.
+ *
+ * @param[in] conn	Pointer to Bluetooth connection (determines used key).
+ * @param[out] out	Pointer to buffer used to store Additional Data packet.
+ * @param[in] data	Data to be encoded to packet.
+ * @param[in] data_len	Length of data (in bytes) to be encoded to packet.
+ * @param[in] nonce	64-bit (8-byte) nonce.
+ *
+ * @return 0 If the operation was successful. Otherwise, a (negative) error code is returned.
+ */
+int fp_keys_additional_data_encode(const struct bt_conn *conn, uint8_t *out, const uint8_t *data,
```

Comment:
@MarekPieta we should discuss it.

Reply:
The module actually handles Fast Pair Key and state of the Fast Pair Procedure. We could rename it, because of that. I would not do it in scope of this PR anyway.